### PR TITLE
Removed the NavMesh rebake step for getting started

### PR DESCRIPTION
**Pull request by ElleEss**
Shuffled around the getting started steps.

Fix in: https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/110

#### Description
Removed NavMesh section

#### Tests
Improbadoc (see screen grabs)

#### Documentation
Is docs.

**Did you remember a changelog entry?**
I wish we had this for docs.

#### Primary reviewers
@Rob, Martijn, Jess

TOC
screen shot 2019-02-15 at 1 44 20 pm

bottom of step 1
screen shot 2019-02-15 at 1 44 34 pm

### CODE REVIEW

**Comment by jamiebrynes7** [comment]
For future readers - the rebaking was fixed in: https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/110

File: docs/content/get-started/set-up.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Get started: 1 - Set up
-There are three parts to this step: 
+There are four parts to this step:
```
Comment (jamiebrynes7) [inline docs/content/get-started/set-up.md]: I love how the PR title is "Remove step" and yet we've ended up incrementing this 😂

File: docs/content/get-started/set-up.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
 # Get started: 1 - Set up
-There are three parts to this step: 
+There are four parts to this step: 
 
 * Sign up for a SpatialOS account (or make sure you are logged in)
 * Set up your machine
 * Get the GDK and the FPS Starter Project source code
+* Open the FPS Starter Project in your Unity Editor
 
 (This step is the longest of the 6 steps - the others are much quicker.)
```
Comment (NotRobSheridan) [inline docs/content/get-started/set-up.md]:
```suggestion
(This step is the longest of the 5 steps - the others are much quicker.)
```

Comment (NotRobSheridan) [inline docs/content/get-started/set-up.md]: But now there are only 5 steps, and there are only 4 steps on this page? Which steps are we talking about? 
```suggestion
Note: Set up is the longest part of this get started guide. The other parts of this guide are much quicker
```
Or something? I'm not sure I like this line at all if I'm being honest.

File: docs/content/get-started/set-up.md (hunk 1 above)
Comment (ElleEss) [reply docs/content/get-started/set-up.md]: Part, Jamie, part. Not a step. Obvs very different.

File: docs/content/get-started/set-up.md (hunk 2 above)
Comment (ElleEss) [reply docs/content/get-started/set-up.md]: Good catch. See this and rejigging slightly.
(Reason for it is: Getting starting with the GDK is marketed as quick and simple...but the first thing you hit is this whopper of a step which is potentially off-putting. From memory, we popped in the NavMesh step temporarily.)

Comment (ElleEss) [inline docs/content/get-started/set-up.md]:
```suggestion
(This page is the longest of the get started guide - the others are much quicker.)
```

Comment (ElleEss) [inline docs/content/get-started/set-up.md]:
```suggestion
(This page is the longest of the Get started guide - the others are much quicker.)
```

Comment (NotRobSheridan) [reply docs/content/get-started/set-up.md]: lgtm

**Pull request merged**